code/game/objects/items/ego_weapons/he.dm

```diff
@@ -744,3 +744,48 @@
 	source.adjustBruteLoss(-10)
 	source.adjustSanityLoss(-5)
 	..()
+
+/obj/item/ego_weapon/impending_day
+	name = "impending day"
+	desc = "You, too, will be a live offering."
+	icon_state = "impending_day"
+	special = "This weapon will attack all non-humans in an AOE after killing a target."
+	force = 55
+	attack_speed = 2
+	hitsound = 'sound/abnormalities/doomsdaycalendar/Doomsday_Attack.ogg'
+	damtype = BLACK_DAMAGE
+	armortype = BLACK_DAMAGE
+	attack_verb_continuous = list("bashes", "clubs")
+	attack_verb_simple = list("bashes", "clubs")
+	attribute_requirements = list(
+							TEMPERANCE_ATTRIBUTE = 40
```

Comment:
Might want to change this to fortitude or prudence, in accordance to the new temperance rebalance. Or maybe not. Talk to Kirie about it.